lib/src/reactive_ble.dart

```diff
@@ -79,13 +64,43 @@ class FlutterReactiveBle {
           .map(const ProtobufConverter().characteristicValueFrom)
             ..listen((_) {});
 
+  final _methodChannel = const MethodChannel("flutter_reactive_ble_method");
+
+  BleStatus _status = BleStatus.unknown;
+
+  final Stream<BleStatus> _statusStream =
+      const EventChannel("flutter_reactive_ble_status")
+          .receiveBroadcastStream()
+          .cast<List<int>>()
+          .map((data) => pb.BleStatusInfo.fromBuffer(data))
+          .map(const ProtobufConverter().bleStatusFrom);
+
+  PrescanConnector _prescanConnector;
+
+  Future<void> _trackStatus() async {
+    await initialize();
+    _statusStream.listen((status) => _status = status);
+  }
+
+  final Stream<ScanResult> _scanStream =
+      const EventChannel("flutter_reactive_ble_scan")
+          .receiveBroadcastStream()
+          .cast<List<int>>()
+          .map((data) => pb.DeviceScanInfo.fromBuffer(data))
+          .map(const ProtobufConverter().scanResultFrom);
+
   final SerialDisposable<Repeater<DiscoveredDevice>> _scanStreamDisposable =
       SerialDisposable((repeater) => repeater.dispose());
 
   Future<void> _initialization;
 
   ScanSession _currentScan;
 
+  /// Initializes the BLE client on native level.
+  ///
+  /// Use this method to explicitly initialize the client. It is not required
+  /// to call initialize before starting a ble operation, the [FlutterReactiveBle]
+  /// make sure that it is initialized before executing the operation.
```

Comment:
I'd recommend _against_ encouraging people to use both `initializa()` and `deinitialize()` methods. In fact, I'd make them private. Will update the doc-comments to avoid encouragement.